biojava-structure/src/main/java/org/biojava/nbio/structure/align/ce/CECalculator.java

```diff
@@ -57,8 +59,9 @@
  */
 public class CECalculator {
 
+	private static final Logger LOGGER = LoggerFactory.getLogger(CECalculator.class);
+
 	protected static final boolean isPrint = false;
```

Comment:
It seems like `isPrint` was also intended to be a flag for debug code. The fact that it is false indicates to me that it should use a log level of at least DEBUG. So maybe the `debug` flag should map to the TRACE level.